cypress/automated-tests/senior/successful-senior-submission.spec.js

```diff
@@ -0,0 +1,232 @@
+    const applicantCity = faker.address.city();
+    const applicantZipCode = '99999';
+
+    cy
+      .get('[data-field-code="CardDeliveryMethod"]')
+      .contains('Mail to my address')
+      .click();
+    cy.get('#element22').type(applicantStreetAddress).blur();
+    cy.get('#element24').type(applicantCity).blur();
+    cy.get('#element26').type(applicantZipCode).blur();
+    cy
+      .get('[data-field-code="MailingAddressBoolean"]')
+      .contains('Yes')
+      .click();
+    cy
+      .get('#form-section-7 > .form-section-buttons > .form-section-next')
+      .click();
+  });
+
+  it("skips the demographic section and agrees to the rules", function() {
+    cy
+      .get('#form-section-8 > .form-section-buttons > .form-section-next')
+      .click();
+    cy
+      .get('[data-field-code="RulesAndConditionsCheckbox"]')
+      .contains('I agree')
+      .click();
+    cy
+      .get('#form-element-wrapper_2 > div > .form-submit-button')
+      .click();
+
+    cy
+      .get('#thank-you-text', { timeout: 15000 })
+      .should('contain', 'Application Submitted');
+  });
+});
+
+describe('youth pass successful submission - without an email address', () => {
+  const faker = require('faker');
+  const applicantZipCode = '99999';
+
+  it("loads the senior form", function() {
+    const seniorUrl = Cypress.env('senior_url');
+    cy.visit(seniorUrl);
+  });
+
+  it('completes the language selection', () => {
+    cy
+      .get('[data-field-code="LanguageSelection"]')
+      .contains('English')
+      .click();
+    cy
+      .get('#form-section-0 > .form-section-buttons > .form-section-next')
+      .click();
+  });
+
+  it("selects apply for a new card", function() {
+    cy
+      .get('[data-field-code="ApplyOrRenewRadio"]')
+      .contains('Apply for a Senior CharlieCard')
+      .click();
+    cy
+      .get('#form-section-1 > .form-section-buttons > .form-section-next')
+      .click();
+  });
+
+  it("clicks through instructions", function() {
+    cy
+      .get('#form-section-2 > .form-section-buttons > .form-section-next')
+      .click();
+  });
+
+  it("enters personal information", function() {
+    const applicantBirthdate = getRandomApplicantAge().applicantBirthdate65to90;
+    const applicantFirstName = faker.name.firstName();
+    const applicantLastName = faker.name.lastName();
+
+    cy.get('#element10').type(applicantBirthdate).blur();
+    cy.get('#element12').type(applicantFirstName).blur();
+    cy.get('#element13').type(applicantLastName).blur();
+    cy
+      .get('#form-section-3 > .form-section-buttons > .form-section-next')
+      .click();
+  });
+
+  it("does not enter an email address or phone number for contact info", function() {
+    cy
+      .get('#form-section-4 > .form-section-buttons > .form-section-next')
+      .click();
+  });
+
+  it("uploads a photo ID", function() {
```

Comment:
Suggestion: What do you think about testing the renewal flow as part of the no email test? We could then ensure that the application can submit successfully without having to upload Photo ID or a Rider Photo.

Reply:
Great suggestion, thanks. I made that change.